starknet_py/tests/e2e/fixtures/misc.py

```diff
@@ -94,6 +99,47 @@ def get_tx_status_full_node_client():
     return f"{FullNodeClient.__module__}.FullNodeClient.get_transaction_status"
 
 
+class ContractVersion(Enum):
+    V1 = "V1"
+    V2 = "V2"
+
+
+class UnkonwnArtifacts(BaseException):
+    pass
+
+
+def load_contract(contract_name: str, version: Optional[ContractVersion] = None):
+    if version is None:
+        if "--contract_dir=v1" in sys.argv:
+            version = ContractVersion.V1
+        if "--contract_dir=v2" in sys.argv:
+            version = ContractVersion.V2
+
+    artifacts_path = CONTRACTS_V2_ARTIFACTS
+    directory = CONTRACTS_V2_COMPILED
+
+    if version == ContractVersion.V1:
+        artifacts_path = CONTRACTS_V1_ARTIFACTS
+        directory = CONTRACTS_V1_COMPILED
+
+    loaded_artifcats = json.loads((artifacts_path).read_text("utf-8"))
+
+    for item in loaded_artifcats["contracts"]:
+        if item["contract_name"] == contract_name:
+            artifacts = item["artifacts"]
+            break
```

Comment:
Let's try to write this in a more idiomatic way for Python. The suggestion below is one option, but there may be better ones.
```suggestion
    artifacts = next((item["artifacts"] for item in loaded_artifacts["contracts"] if item["contract_name"] == contract_name), None)
```